ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs

```diff
@@ -91,6 +95,88 @@ public void Build()
+            {
+                var listType = new List<Type>();
+                item.Types.ForEach(type =>
+                {
+                    item.Types.ForEach(m => {
+                        var classMonikers = m.BaseTypes.Where(bt => bt.Name == "System.Object");
+                        enumCovertClassMoniker = classMonikers.SelectMany(o => o.Monikers).ToList();
+                    });
+
+                    if (_frameworks == null && _frameworks.DocIdToFrameworkDict == null)
+                    {
+                        return;
+                    }
+
+                    ///Generate Enum Type
+                    var enumType = type.DeepCopy();
+                    var suffix = "_" + type.ItemType.ToString().Substring(0, 1).ToLower();
+                    enumType.ItemType = ItemType.Enum;
+                    enumType.Id = enumType.Id + suffix;
+                    enumType.FullName = type.FullName + suffix;
+                    var monikers = _frameworks.DocIdToFrameworkDict[enumType.DocId].ToList();
+                    var enumItemDocId = enumType.DocId + suffix;
+                    enumType.DocId = enumItemDocId;
+                    monikers = monikers.Except(enumCovertClassMoniker).ToList();
+                    _frameworks.DocIdToFrameworkDict[enumItemDocId] = monikers;
+                    var members = new List<Member>();
+                    foreach (var m in enumType.Members)
+                    {
+                        if (!_frameworks.DocIdToFrameworkDict[m.DocId].Any(moniker => enumCovertClassMoniker.Contains(moniker)))
+                        {
+                            members.Add(m);
+                        }
+                    }
+
+                    enumType.Members = members;
+                    members = new List<Member>();
+                    foreach (var m in enumType.Overloads)
+                    {
+                        if (!_frameworks.DocIdToFrameworkDict[m.DocId].Any(moniker => enumCovertClassMoniker.Contains(moniker)))
+                        {
+                            members.Add(m);
+                        }
+                    }
+
+                    enumType.Overloads = members;
+                    listType.Add(enumType);
+
+                    ///Generate Class Type
+                    members = new List<Member>();
+                    type.ItemType = ItemType.Class;
+                    _frameworks.DocIdToFrameworkDict[type.DocId] = enumCovertClassMoniker;
+                    foreach (var m in type.Members)
+                    {
+                        if (_frameworks.DocIdToFrameworkDict[m.DocId].Any(moniker => enumCovertClassMoniker.Contains(moniker)))
+                        {
+                            members.Add(m);
+                        }
+                    }
+
+                    type.Members = members;
+                    members = new List<Member>();
+                    foreach (var m in type.Overloads)
```

Comment:
Overloads is not built yet at this point, so you can ignore them now. An enum type won't need overloads anyway.